src/Form/Type/ContainerTemplateType.php

```diff
@@ -58,14 +57,6 @@ public function getParent()
         return ChoiceType::class;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
```

Comment:
Isn't this a BC break?

Reply:
Yes, because symfony provides this method itself

Reply:
https://github.com/symfony/symfony/blob/v2.8.0/src/Symfony/Component/Form/AbstractType.php#L47